web/sites/default/config/filter.format.full_html.yml

```diff
@@ -62,14 +68,14 @@ filters:
+      allowed_html: '<br> <p class="indent-1 indent-2 indent-3 indent-4 indent-5 lead text-align-left text-align-center text-align-right text-align-justify"> <h1 class="text-align-left text-align-center text-align-right text-align-justify"> <h2 id class="h6 text-align-left text-align-center text-align-right text-align-justify"> <h3 id class="text-align-left text-align-center text-align-right text-align-justify"> <h4 id class="text-align-left text-align-center text-align-right text-align-justify"> <h5 id class="text-align-left text-align-center text-align-right text-align-justify"> <h6 id class="text-align-left text-align-center text-align-right text-align-justify"> <a class="btn-cta btn-cta-outline nav-link active" hreflang href data-entity-type data-entity-uuid data-entity-substitution> <ol class="code-list code-section-list" type start> <ul class="checkbox-list nav nav-group-menu" type> <cite> <address> <drupal-entity data-aspect-ratio data-alt-link-text data-align data-start-cue alt title data-caption data-entity-embed-display data-entity-embed-display-settings data-view-mode data-entity-uuid data-langcode data-embed-button="audio_video_browser document_browser image_browser insert_content_fragment insert_iframe map_browser" data-entity-type="media node"> <img alt src> <blockquote cite> <li class="nav-item"> <strong> <em> <code class="language-*"> <pre class="text-align-left text-align-center text-align-right text-align-justify"> <s> <sub> <sup> <hr> <table> <tr> <td rowspan colspan> <th rowspan colspan> <thead> <tbody> <tfoot> <caption> <dl class> <dt> <dd>'
```

Comment:
For the reviewer: I'm not sure why `<li class="code-alpha-fix code-alpha3-fix nav-item">` is changed into `<li class="nav-item">`. The only config change is to add the custom accordion heading filter.